.travis.yml

```diff
@@ -34,5 +34,15 @@ script:
 after_success:
     - if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then pip install coveralls --use-mirrors ; coveralls ; fi
 
+# publish new version to pypi
+deploy:
+  provider: pypi
+  skip_existing: true
+  on:
+    tags: true
+  user: $PYPI_USER
+  password: $PYPI_PASSWORD
```

Comment:
The [Travis docs](https://docs.travis-ci.com/user/deployment/pypi/) mention "It is also possible, but not recommended, to use PyPI user and password, instead of token" -- though they don't actually mention the advantage of one over the other.

Reply:
They are more secure because:

* Are longer so they probably take longer to brute force
* And if they are hacked in some way, they are limited to specific project and can be revoked more easily

With this PR, it is possible to use password or token, because when using token you just set username to `__token__` and password to actual token and this can be done with environment variables.